### PR TITLE
Fix laptop view

### LINKED ISSUE #11: Add visual or text indication that prompts interaction
There needs to be some indication (visual or a text prompt) that the user can click on the chart elements to pull up additional information

### LINKED ISSUE #19: Fix display on laptop-sized screen
There are couple of issues w/ the laptop view.
* text size need to be smaller
* The rectangles that trigger interaction are placed differently when the page loads on a laptop-sized screen. I started fixing this -- it has to do with how the chart height is dynamically set. 

Monitor:
image

Laptop:
image

### LINKED ISSUE #21: highlight text instead of row
Text highlighting is not capturing letters directly, just the row they are in, so if a highlighted label has a g or j etc part of the letter is faded out.
image

### PR DESCRIPTION
**Pull request by hcorson-dosch**
This PR fixes #19 and fixes #21 and finalizes the interaction set up, which closes #11

The interaction rectangles are now appended to `this.svgChart` instead of the `this.svg`. As a result they now align with the chart elements on different window sizes.

Now on desktop the rectangles used to dim the chart do not cover the left margin, so do not dim the text. The text is now dimmed directly, per #21. However, I've set it up so the full left margin space is still used to trigger click and mouseover events on desktop.

The PR also adjust the height and width settings for the `chart-container` and restricts the width of the page content so that it's not really wide on huge monitors.

This PR also makes some adjustments to the font-sizing for the text at the top of the page as well as the y-axis text.

### CODE REVIEW

**Comment by ceenell** [comment]
When on `linear` the mouseover is making non-selected rows half-hilighted 
image

The text could still use a little bit more hierarchy. What about increasing the weight of the title and the axis label, or just the words Pool and Flux where they appear? Also the text sizes for the top text and text below the chart are pretty different. The line spacing of the top text seems high compared to the rest of the page.

File: src/App.vue [hunk 1]
```diff
@@ -110,10 +110,16 @@ h3{
   padding-top: .5em;
   font-family: $Assistant;
   font-weight: 300;
```
Comment (ceenell) [inline src/App.vue:112]: What about 700?

File: src/components/pool_flux_chart.vue [hunk 2]
```diff
@@ -511,23 +565,25 @@ export default {
...
   #caption-container {
     display: block;
   }
   .poolText {
     color: $poolColor;
-    font-weight: 500;
   }
   .fluxText {
     color: $fluxColor;
-    font-weight: 500;
```
Comment (ceenell) [inline src/components/pool_flux_chart.vue:530]: I think it could work well to bump the weight a little bit and make "pools" and "fluxes" pop throughout

**Review by ceenell — CHANGES_REQUESTED** [review]
Need to address half moons on desktop mouseover

**Comment by hcorson-dosch** [comment]
> When on `linear` the mouseover is making non-selected rows half-hilighted image

👀 Good catch!! I'll fix that

> The text could still use a little bit more hierarchy. What about increasing the weight of the title and the axis label, or just the words Pool and Flux where they appear? 

Agreed that it needs more refinement. The words pool and flux are higher weight - I'll see if I can go higher.

> Also the text sizes for the top text and text below the chart are pretty different on desktop but look more consistent on mobile. The line spacing of the top text seems high compared to the rest of the page.

👍 I'll make some tweaks

**Comment by hcorson-dosch** [comment]
Ok - changes made. I fixed the half-moon issue, made font sizes more consistent, and upped the emphasis of 'pool' and 'flux' throughout.

File: src/components/pool_flux_chart.vue [hunk 3]
```diff
@@ -229,39 +232,33 @@ export default {
...
-            .attr("text-anchor", "middle")
-            .attr("x", this.chartWidth/2)
-            .attr("y", this.chartHeight + 42)
-            .text("Pool volume (km³) or flux rate (km³ per year)")
+          this.svgChart.append("foreignObject")
+            .attr("id", "x-label-container")
+            .attr("x", 0)
+            .attr("y", this.chartHeight+25)
+            .attr("width", this.chartWidth)
+            .html("<p class='x_label'><span class='poolText emph'>Pool</span> volume (km³) or <span class='fluxText emph'>flux</span> rate (km³ per year)</p>")
```
Comment (ceenell) [inline src/components/pool_flux_chart.vue:240]: 🆒

**Review by ceenell — APPROVED** [review]
Great - everything ran for me locally and it's coming together.

**Pull request merged**